hack/local-development/common/local-imagevector-overwrite

```diff
@@ -35,15 +43,15 @@ cleanup_imagevector_overwrite() {
 
 compute_imagevector_overwrite() {
   imagevector_overwrite="images: []"
```

Comment:
Ah, I didn't see that this is another function, sorry. I'm not sure whether it now just works by accident. Please also define them here, just to be safe:

```suggestion
  REPO_ROOT="$1"
  CHARTS_DIRECTORY="$2"

  imagevector_overwrite="images: []"
```

Reply:
tested it out and it apparently works - even in the extension. 
I added it to make it more understandable.